draft-ietf-quic-transport.md

```diff
@@ -2357,11 +2361,11 @@ shown in the following figure and described below.
                             +--------+
                                  |
                         send/recv STREAM/RST
-                             recv MSD/SB
+                             recv MSD/SB             
                                  |
                                  v
-                 recv FIN/  +--------+    send FIN/
-                 recv RST   |        |    send RST
+              recv FIN/     +--------+     send FIN/
+         recv RST/send UNI  |        |  send RST/recv UNI
```

Comment:
maybe uni bit on RST stream frame as well?
ya I think it needs an additional state after send UNI technically.